libkbfs/delayed_cancellation_test.go

```diff
@@ -134,14 +134,14 @@ func TestDelayedCancellationEnabled(t *testing.T) {
 	t.Parallel()
 
 	ctx, cancel := makeContextWithDelayedCancellation(t)
-	EnableDelayedCancellationWithGracePeriod(ctx, 15*time.Millisecond)
+	EnableDelayedCancellationWithGracePeriod(ctx, 50*time.Millisecond)
 
 	cancel()
 
 	select {
 	case <-ctx.Done():
 		t.Fatalf("Cancellation is not delayed")
-	case <-time.After(10 * time.Millisecond):
+	case <-time.After(20 * time.Millisecond):
```

Comment:
Can we keep this at 10 to reduce the probability of a flake?  Or are you afraid this would be too likely to give a false positive if delayed cancellation is every actually broken?